src/components/Task/Task.js

```diff
@@ -0,0 +1,25 @@
+import React, { Component } from 'react'
```

Comment:
Always look at the newborn component inside one more component and try to separate this logic like the granular way of writing app